src/pipeline.cpp

```diff
@@ -137,7 +155,11 @@ void openxc::pipeline::publish(openxc_VehicleMessage* message,
 void openxc::pipeline::sendMessage(Pipeline* pipeline, uint8_t* message,
         int messageSize, MessageClass messageClass) {
     sendToUsb(pipeline, message, messageSize, messageClass);
-    sendToUart(pipeline, message, messageSize, messageClass);
+	#ifdef TELIT_HE910_SUPPORT
```

Comment:
I see now as I read through, it's taking over the UART communication to talk to the modem. I feel like that could be more clear in the code, but I'm not sure how.